common-flow.md

```diff
@@ -56,8 +60,16 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
        release/production" state to reduce any friction with creating a new
        release.
 3. Change Branches
-    1. Each change (feature, bugfix, etc.) MUST be performed on separate
-       branches that SHOULD be referred to as "change branches".
+    1a. Before a project is Live;
```

Comment:
The 1a and 1b appear to have messed up the markdown formatting. Perhaps you need a nested list within 1.